RESTOpenSubtitlesHandler/Models/Responses/SearchResult.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.Collections.Generic;
+
+namespace RESTOpenSubtitlesHandler.Models.Responses
+{
+    public class SearchResult
+    {
+        public int TotalPages;
+        public int TotalCount;
+        public string Page;
```

Comment:
opensubtitles may change this to an int so I'm leaving this here until they respond

Reply:
You can probably make it in an int and C# will probably try to automatically parse it already